deadspace/code/necromorph/abilities/long_charge.dm

```diff
@@ -71,17 +71,17 @@
 		if(charger.dir != direction) //It needs to move twice in the same direction, at least, to begin charging.
 			return
 		charge_dir = direction
-		if(!check_momentum(direction))
+		if(!check_momentum(direction))	
 			charge_dir = null
 			return
 		charger.charging = CHARGE_BUILDINGUP
 		handle_momentum()
 		spec_check(source, oldloc, direction, Forced, old_locs)
 		return
 
-	if(!check_momentum(direction))
-		do_stop_momentum()
-		return
+	//if(!check_momentum(direction))
+	//	do_stop_momentum()
+	//	return
```

Comment:
Commenting out code isn't how you fix code. If it should be removed, remove it. 

Whether or not it actually needs to be removed is @DTraitor 's call.

Reply:
Got it, I've merged some new changes that remove this and change the line playsound is on.